doc/changes/devel/12727.newfeature.rst

```diff
@@ -0,0 +1,3 @@
+:func:`mne.minimum_norm.make_inverse_operator` and related functions now more robustly
+warn if the whitener computed from the noise covariance has an incorrect rank,
+by `Eric Larson`_.
```

Comment:
This change showed a real bug we had, glad you suggested we add it @hoechenberger !

Reply:
amazing, thanks for addressing this!